Dockerfile.erb

```diff
@@ -73,6 +73,19 @@ ADD templates/logstash.config.erb /logstash.config.erb
 ADD templates/log4j.properties /log4j.properties
 ADD bin/run-gentleman-jerry.sh run-gentleman-jerry.sh
 
+# Install BATS (for tests)
+RUN curl -sL https://github.com/sstephenson/bats/archive/master.zip > /tmp/bats.zip \
+    && cd /tmp \
+    && unzip -q bats.zip \
+    && ./bats-master/install.sh /usr/local \
+    # /usr/local/bin/bats should be a symlink, so make it happen.
+    && ln -sf /usr/local/libexec/bats /usr/local/bin/bats \
+    && rm -rf /tmp/bats*
+
+# Install openssl (needed for tests)
+# REVIEW: Is openssl client needed for anything else, or can we remove it after tests?
+RUN apk add openssl
```

Comment:
OpenSSL is definitely used by `stunnel` (see: `bin/install-stunnel.sh`), but we don't necessarily the CLI tools — just the runtime library. It appears [the `openssl` package is ONLY the CLI tools](https://pkgs.alpinelinux.org/package/v3.7/main/x86_64/openssl), so we should be fine to install it only for the tests, like we do with the JDK in `run_tests.sh`